docs/content/reference/ransomware.md

```diff
@@ -6,14 +6,22 @@ pre: '<i class="fas fa-lock"></i> '
 weight: 10
 ---
 
-The Infection Monkey has the capability of simulating a ransomware attack on your network.
+The Infection Monkey has the capability of simulating a ransomware attack on your network through a series of activities.
+
+#### Encrypting user-specified files
 All actions performed by the encryption routine are designed to be safe for production
 environments.
 
 To ensure minimum interference and easy recoverability, the ransomware simulation will encrypt
 files only if the user specifies a directory that contains files that are safe to encrypt.
 If no directory is specified, no files will be encrypted.
 
+#### Leaving a README.txt file
+If a target directory is specified for the encryption routine, the ransomware simulation can be configured to leave a README.txt file there. This file clearly states that there is no need to panic and only a simulation is taking place.
+
+The contents of the file can be found [here](https://github.com/guardicore/monkey/tree/develop/monkey/infection_monkey/ransomware/ransomware_readme.txt).
```

Comment:
I think we should give the user some context about why we leave a README. Below are my suggestions.

```suggestion
Many ransomware packages leave a README.txt file on the victim machine with an explanation of what has occurred and instructions for paying the attacker. Infection Monkey can also leave a README.txt file on the victim machine in order to replicate this behavior. This can be enabled or disabled by checking the box in the configuration screen. Note that if no target directory is specified, Infection Monkey will not leave a README.txt file.

<!-- add screenshot highlighting readme option -->

The README.txt file informs the user that a ransomware simulation has taken place and that they should contact their administrator. The contents of the file can be found [here](https://github.com/guardicore/monkey/tree/develop/monkey/infection_monkey/ransomware/ransomware_readme.txt).
```